spec/src/lib.rs

```diff
@@ -237,23 +237,23 @@ impl GenesisCell {
     fn build_output(&self) -> Result<CellOutput, Box<dyn Error>> {
         let mut cell = CellOutput::default();
         cell.data = self.message.as_bytes().into();
-        cell.lock.clone_from(&self.lock);
```

Comment:
Is the `into` needed?

Reply:
They have different types, one is core script, another is jsontypes script